APINetCore/src/ApiNetCore.Application/DTOs/Validations/BusinessRulesValidators/BusinessRulesValidator.cs

```diff
@@ -0,0 +1,32 @@
+﻿using ApiNetCore.Application.DTOs.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ApiNetCore.Application.DTOs.Validations.BusinessRulesValidators
+{
+    public class BusinessRulesValidator : IBusinessRules

```

Comment:
Those double validations could be modified to return their own message each. So instead of a validation method return true or false, it could return a List of String containing all invalid messages.